casa/System/AppState.h

```diff
@@ -65,6 +65,14 @@ class AppState {
         return result;
     }
 
+    // get directory containing IERS measures data
+    // an exception is thrown if it does not exist or
+    // does not contain the IERS tables
+    virtual std::string measuresDir( ) const {
+        static std::string result;
+        return result;
```

Comment:
This function in the current form does not make sense, is it overridden by a CASA specific override?

Reply:
Yes, CASA derives a class from AppState which includes more configuration settings (that casacore does not care about) and then registers an object created from that derived class for use as the AppState object. casacore gets the base class object and casa gets the derived class object.